docs/understanding-results.md

```diff
@@ -0,0 +1,248 @@
+  "initialUrl": "http://example.com",
+  "url": "https://www.example.com/",
+  "score": 50,
+  "audits": {...},
+  "runtimeConfig": {...},
+  "timing": {...},
+  "reportCategories": [{...}],
+  "reportGroups": {...},
+}
+```
+
+
+<a name="audits"></a>
+## `audits`
+
+An object containing the results of the audits, keyed by their name.
+
+### Audit Properties
+| Name | Type | Description |
+| -- | -- | -- |
+| name  | `string` | The string identifier of the audit in kebab case.  |
+| category | `string` | No longer used. *WARNING: Deprecated, will be removed in Lighthouse 3.0* |
+| description | `string` | The brief description of the audit's successful state. |
+| helpText | `string` | A more detailed description that describes why the audit is important and links to Lighthouse documentation on the audit, markdown links supported. |
+| debugString | <code>string&#124;undefined</code> | A string indicating some additional information to the user explaining an unusual circumstance or reason for failure. |
+| informative | `boolean` | Indicator used for display that the audit is intended to be informative only, not failable. |
+| manual | `boolean` | Indicator used for display that the audit does not have results and is a placeholder for the user to conduct manual testing. |
+| score | <code>boolean&#124;number</code> | The scored value determined by the audit as either boolean or a number `0-100`. If the audit is a boolean, the implication is `score ? 100 : 0`. |
+| rawValue | <code>boolean&#124;number</code> | The unscored value determined by the audit. Typically this will match the score if there's no additional information to impart. For performance audits, this value is typically a number indicating the metric value. |
+| displayValue | `string` | The string to display in the report alongside audit results. If empty, nothing additional is shown. This is typically used to explain additional information such as the number and nature of failing items. |
+| scoringMode | <code>"binary"&#124;"numeric"</code> | A string identifying how granular the score is meant to be indicating, i.e. is the audit pass/fail or are there shades of gray 0-100. *NOTE: This does not necessarily mean `typeof audit.score === audit.scoringMode`, an audit can have a score of 40 with a scoringMode of `"binary"` meant to indicate display should be failure.* |
+| details | `Object` | Extra information found by the audit necessary for display. The structure of this object varies from audit to audit. The structure of this object is somewhat stable between minor version bumps as this object is used to render the HTML report.
+| extendedInfo | `Object` | Extra information found by the audit. The structure of this object varies from audit to audit and is generally for programmatic consumption and debugging, though there is typically overlap with `details`. *WARNING: The structure of this object is not stable and cannot be trusted to follow semver* |
```

Comment:
can you change the order of the properties here?

name
description
helpText
debugString
rawValue
displayValue
score
scoringMode
details
extendedInfo
manual
informative
category

Reply:
sure, done